types/src/crypto/signatures/signable.rs

```diff
@@ -94,9 +91,6 @@ impl AsRef<[u8]> for Signature {
 pub struct EncryptedSignatureCtype;
 pub type EncryptedSignature = Ciphertext<EncryptedSignatureCtype>;
 
-impl EarEncryptable<IdentityLinkKey, EncryptedSignatureCtype> for Signature {}
```

Comment:
I believe `EncryptedSignature` is also unused and can be removed.

Reply:
Thanks. Removed.